### ISSUE TITLE
fix: not compatible with the new version of Flutter

### ISSUE THREAD

**Issue by arturograu** [VeryGoodOpenSource/very_good_cli#738]
**Description**
Looks like very_good_cli is not compatible with the latest Flutter version.
When creating an app project for the first time I get this error:
```
✓ Generated 240 file(s) (0.2s)
✓ Running "flutter packages get" in ./my_app (1.1s)
Unhandled exception:
ProcessException: Standard out
Resolving dependencies...
Standard error
Because my_app depends on flutter_localizations from sdk which depends on intl 0.18.0, intl 0.18.0 is required.
So, because my_app depends on intl ^0.17.0, version solving failed.
  Command: flutter packages get
#0      _Cmd._throwIfProcessFailed (package:very_good_cli/src/cli/cli.dart:145:7)
#1      _Cmd.run (package:very_good_cli/src/cli/cli.dart:97:7)
<asynchronous suspension>
#2      Flutter.packagesGet.<anonymous closure> (package:very_good_cli/src/cli/flutter_cli.dart:101:11)
<asynchronous suspension>
#3      _runCommand (package:very_good_cli/src/cli/flutter_cli.dart:304:13)
<asynchronous suspension>
#4      Flutter.packagesGet (package:very_good_cli/src/cli/flutter_cli.dart:87:5)
<asynchronous suspension>
#5      installFlutterPackages (package:very_good_cli/src/commands/create/templates/post_generate_actions.dart:28:5)
<asynchronous suspension>
#6      VeryGoodCoreTemplate.onGenerateComplete (package:very_good_cli/src/commands/create/templates/very_good_core/very_good_core_template.dart:20:5)
<asynchronous suspension>
#7      CreateSubCommand.runCreate (package:very_good_cli/src/commands/create/commands/create_subcommand.dart:226:5)
<asynchronous suspension>
#8      CreateSubCommand.run (package:very_good_cli/src/commands/create/commands/create_subcommand.dart:200:20)
<asynchronous suspension>
#9      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#10     VeryGoodCommandRunner.runCommand (package:very_good_cli/src/command_runner.dart:211:18)
<asynchronous suspension>
#11     VeryGoodCommandRunner.run (package:very_good_cli/src/command_runner.dart:148:14)
<asynchronous suspension>
#12     main (file:///Users/arturograu/.pub-cache/hosted/pub.dev/very_good_cli-0.14.0/bin/very_good.dart:5:24)
```

**Steps To Reproduce**

1. Make sure your Flutter version is 3.10.0
2. Run very_good create flutter_app my_app --desc "My App"

**Expected Behavior**
The command should execute successfully creating a new Flutter project.

**Comment by fjugaldev** [comment]
Hi guys and then there another bug related with the Missing RunnerTest config folder for iOS platform

**Comment by renancaraujo** [comment]
Yeah, we are upgrading everything to Dart 3 soon. 🐻 with me there are lots fo code to update

**Comment by renancaraujo** [comment]
Yeah, just released 0.15.0. This should be fixed.

**Comment by icodeyou** [comment]
I had the same issue running `very_good create flutter_app test`
If you did this, just don't use `test` as a name for your app.
